literature.bib

```diff
@@ -12106,11 +12109,11 @@ @inproceedings{LK+:ECBS07
 }
 
 @article{NCB+SEI07,
-  	author={Northrop, Linda and Clements, P.},
-  	title={{A Framework for Software Product Line Practice, Version 5.0}},
- 	 journal={SEI},
-	address =  {Pittsburgh,USA},
-  	year=2007
+  	author = {Northrop, Linda and Clements, P.},
+  	title = {{A Framework for Software Product Line Practice, Version 5.0}},
+ 	 journal = {SEI},
```

Comment:
brackets too much?

Reply:
Again, not mine, just fixed spaces, but will fix it, too.

Reply:
This entry is originally from @tthuem, won't touch.